### PR TITLE
Create dankamigos.json

### PR DESCRIPTION
**Pull request by mokshdk**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ `dankamigos.json` ] The file is in the `domains` folder and is in the JSON format.
- [  yes ] The file's name is all lowercased and alphanumeric.
- [  yes  ] You have completed your website.
- [  yes ] The website is reachable.
- [  yes ] The CNAME record doesn't contain `https://` or `/`. 
- [ yes ] There is sufficient information at the `owner` field.  

## Website Link/Preview
https://dankamigos-baapekd9aua9esh2.eastus-01.azurewebsites.net/

### CODE REVIEW

File: domains/dankamigos.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "mokshdk",
+    "email": "anirudhsus001@gmail.com"
+  },
+  "record": {
+    "A": ["20.119.16.40"],
+    "TXT": ["562965D131E7C913D1796476301A86A148A2B64B52D99FC316724B0B785FAAB1"],
+    "CNAME": "dankamigos-baapekd9aua9esh2.eastus-01.azurewebsites.net"
```
Comment (Stef-00012) [inline domains/dankamigos.json:9]: You can't have CNAME along with other records

**Review by Stef-DP — DISMISSED** [review]

**Comment by mokshdk** [comment]
okie

**Comment by mokshdk** [comment]
bymistake i closed it 🤦

File: domains/dankamigos.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "mokshdk",
+    "email": "anirudhsus001@gmail.com"
+  },
+  "record": {
+    "A": ["20.119.16.40"],
+    "TXT": ["562965D131E7C913D1796476301A86A148A2B64B52D99FC316724B0B785FAAB1"],
```
Comment (CuteDog5695) [inline domains/dankamigos.json:8]:
```suggestion
    "TXT": ["562965D131E7C913D1796476301A86A148A2B64B52D99FC316724B0B785FAAB1"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Now it was fine, dunno why it took 4 days to actually merge it.

**Pull request merged**